core/driver/tests/src/test/scala/org/virtuslab/ideprobe/ProbeDriverTest.scala

```diff
@@ -26,11 +26,14 @@ import org.virtuslab.ideprobe.robot.RobotPluginExtension
 
 @RunWith(classOf[JUnit4])
 final class ProbeDriverTest extends IdeProbeFixture with Assertions with RobotPluginExtension {
-  private val intelliJProvider = IntelliJProvider.Default
```

Comment:
you should be able to keep the original code

Reply:
fixed as suggested